.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+## Summary
```

Comment:
Please don't push `.github`, `.Ds_store`  and those king of files please

Reply:
I hadn't noticed, they are already included in .gitignore